doc/chomp_interface/chomp_interface_tutorial.rst

```diff
@@ -70,3 +70,57 @@ To run the CHOMP planner with obstacles, do the following in two seperate termin
 
   roslaunch panda_moveit_config demo_chomp.launch
   python collision_scene_test_1.py OR python collision_scene_test_2.py
+
+
+Tweaking some of the parameters for CHOMP
+-----------------------------------------
+Chomp has some optimization parameters associated with it. These can be modified for the given environment/robot you are working with and is normally present in the `chomp_planning.yaml <https://github.com/ros-planning/panda_moveit_config/blob/master/config/chomp_planning.yaml>`_ file in config folder of the robot you are working with. If this file does not exist for your robot, you can create it and set the parameter values as you want. Following are some of the insights to set up these parameter values for some of them:
```

Comment:
Chomp -> "CHOMP"

"The following are some..."

Reply:
done